ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKGroupedStreamTest.java

```diff
@@ -178,7 +177,7 @@ public void shouldBuildStepForWindowedAggregate() {
             ExecutionStepFactory.streamWindowedAggregate(
                 queryContext,
                 schemaGroupedStream.getSourceStep(),
-                Formats.of(expected, valueFormat, SerdeOption.none()),
+                io.confluent.ksql.execution.plan.Formats.of(expected, valueFormat, SerdeOption.none()),
```

Comment:
hummm... nasty name clash... can we rename one? e.g. `PlanFormats`? Or make your `Formats` something like `FormatFactory`?

Reply:
renamed mine to `FormatFactory` because the other is in the serialized plan